### PR TITLE
Switch slack.k8s.io to the community inviter link

### PR DESCRIPTION
**Pull request by dims**
related to https://github.com/kubernetes/k8s.io/issues/4099

Signed-off-by: Davanum Srinivas <davanum@gmail.com>

### CODE REVIEW

**Comment by ameukam** [comment]
/lgtm
/hold

Remember that certificates needs to be manually deployed.

**Comment by ameukam** [comment]
It would be safe if we deploy the DNS change first before we deploy the certificate.

**Comment by ameukam** [comment]
Deployed to canary:

```bash
ameukam@cloudshell:~/k8s.io/apps/k8s-io (kubernetes-public)$ kubectl -n k8s-io-canary apply -f configmap-nginx.yaml 
configmap/nginx configured
```

File: apps/k8s-io/configmap-nginx.yaml [hunk 1]
```diff
@@ -325,6 +325,13 @@ data:
         rewrite ^/(.*)?/source$  https://storage.googleapis.com/kubernetes-release/release/$1/kubernetes-source.spdx redirect;
       }
 
+      server {
+        server_name slack.k8s.io slack.kubernetes.io
+        listen 80;
+
+        rewrite ^/$ https://communityinviter.com/apps/kubernetes/community redirect;
```
Comment (ameukam) [inline apps/k8s-io/configmap-nginx.yaml:332]:
```suggestion
       location / {
         rewrite ^/.$ https://communityinviter.com/apps/kubernetes/community permanent;
       }
```

**Comment by dims** [comment]
thanks for the suggestion @ameukam - accepted the change.

**Comment by ameukam** [comment]
Change redeployed:

```bash
 curl --header  "Host: slack.k8s.io" -I http://34.102.239.89/
HTTP/1.1 302 Moved Temporarily
Server: nginx
Date: Fri, 19 Aug 2022 12:21:03 GMT
Content-Type: text/html
Content-Length: 138
Location: https://communityinviter.com/apps/kubernetes/community
Via: 1.1 google

❯ curl --header  "Host: slack.kubernetes.io" -I http://34.102.239.89/
HTTP/1.1 302 Moved Temporarily
Server: nginx
Date: Fri, 19 Aug 2022 12:21:17 GMT
Content-Type: text/html
Content-Length: 138
Location: https://communityinviter.com/apps/kubernetes/community
Via: 1.1 google
```

**Comment by dims** [comment]
Nice thanks!

**Pull request merged**